### PR TITLE
Bump psammead story-promo-list

### PR DESCRIPTION
**Pull request by rhenshaw56**
Resolves N/A

**Overall change:**
_Bumps psammead story-promo-list to manually regress changes due to https://github.com/bbc/psammead/pull/3588._

**Code changes:**

- _Bump psammead story-promo-list._
- _Update snapshots._

**Testing notes:**
The section label is used in thee following containers/pages 
- CPS Related content (Stories page)
- CPS Top Stories (Stories / Front pages )
- Front Page story rows (Front page)
- Index Page section (IDX page)

Testing would ideally involve comparing this change in the listed containers/pages with [storybook on latest](https://bbc.github.io/simorgh/) and visually observing that they still look the same way.

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added labels to this PR for the relevant pod(s) affected by these changes
- [X] I have assigned this PR to the Simorgh project

**Testing:**

- [X] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [X] This PR requires manual testing

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -14624,7 +14624,7 @@
     },
     "jest-silent-reporter": {
       "version": "0.2.1",
-      "resolved": "https://registry.npmjs.org/jest-silent-reporter/-/jest-silent-reporter-0.2.1.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/jest-silent-reporter/-/jest-silent-reporter-0.2.1.tgz",
```
Comment (chris-hinds) [inline package-lock.json]: Lots of morph npm registries here.

**Review by chris-hinds — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (rhenshaw56) [reply package-lock.json]: @hindsc52 fixed.

**Review by chris-hinds — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (chris-hinds) [reply package-lock.json]: Thanks @rhenshaw56

**Review by simonsinclair — APPROVED** [review]

**Comment by PriyaKR** [comment]
Looks good to me.

**Pull request merged**